src/EditorFeatures/Core/DocumentationComments/CopilotGenerateDocumentationCommentProvider.cs

```diff
@@ -63,20 +64,24 @@ public async Task GenerateDocumentationProposalAsync(DocumentationCommentSnippet
 
             // Do not do IntelliCode line completions if we're about to generate a documentation comment
             // so that won't have interfering grey text.
-            var intellicodeLineCompletionsDisposable = await _suggestionManager!.DisableProviderAsync(SuggestionServiceNames.IntelliCodeLineCompletions, cancellationToken).ConfigureAwait(false);
+            var intelliCodeLineCompletionsDisposable = await _suggestionManager!.DisableProviderAsync(SuggestionServiceNames.IntelliCodeLineCompletions, cancellationToken).ConfigureAwait(false);
+            var suggestion = new DocumentationCommentSuggestion(this, _suggestionManager, intelliCodeLineCompletionsDisposable);
+            var suggestionSessionStarted = await suggestion.StartSuggestionSessionAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
nit: i feel this unnecessarily exposed the internal of `DocumentationCommentSuggestion`, (like you need to make sure the order of calls, etc.) consider passing in a lambda to calculate the proposal to `StartSuggestionSessionAsync` and let suggestion handles display/dismiss in it.

Reply:
thoughts on this comment Ankita?